### PR TITLE
Issue 1

### PR DESCRIPTION
**Pull request by Wagnesio**

### CODE REVIEW

File: TKCurrencyTextField/Classes/TKCurrencyTextField.swift
Comment (willwfsp) [inline TKCurrencyTextField/Classes/TKCurrencyTextField.swift:62]: Prefer compact code and let the compiler infer the type for constants or variables of single instances.

Ref: [Type Inference](https://github.com/raywenderlich/swift-style-guide#type-inference)

File: TKCurrencyTextField/Classes/TKCurrencyTextField.swift
Comment (willwfsp) [inline TKCurrencyTextField/Classes/TKCurrencyTextField.swift:69]: Prefer compact code and let the compiler infer the type for constants or variables of single instances.

Ref: [Type Inference](https://github.com/raywenderlich/swift-style-guide#type-inference)

File: TKCurrencyTextField/Classes/TKCurrencyTextField.swift
Comment (willwfsp) [inline TKCurrencyTextField/Classes/TKCurrencyTextField.swift]: Please reident your code properly. If possible, use 2 spaces identation. Use [Swift Lint](https://github.com/realm/SwiftLint) to help you on this

File: TKCurrencyTextField/Classes/TKCurrencyTextField.swift
Comment (willwfsp) [inline TKCurrencyTextField/Classes/TKCurrencyTextField.swift:117]: Same tip here. Prefer compact code and let the compiler infer the type for constants or variables of single instances.

Please verify it for whole project
Ref: [Type Inference](https://github.com/raywenderlich/swift-style-guide#type-inference)

File: TKCurrencyTextField/Classes/TKCurrencyTextField.swift
Comment (willwfsp) [inline TKCurrencyTextField/Classes/TKCurrencyTextField.swift:63]: Underscore charater is not necessary. You can use this form:

```swift
((UITextField, NSRange, String) -> ())
```

If type isn't clear itself, create `typealiases`:

```swift
typealias ReplacementString = String

...

((UITextField, NSRange, ReplacementString) -> ())
```

**Review by damboscolo — APPROVED** [review]

**Comment by willwfsp** [comment]
@damboscolo pull request can be merged only after my approval. See the comments below about the code style.

**Comment by Edudjr** [comment]
Hey guys, remember to close the branch when it is merged to keep the repo clean! 
Nice work btw :)

ps: does this merge fix the issue https://github.com/tokenlab/TKCurrencyTextField/issues/4 ?

**Pull request merged**
